stdlog/stdlog-intern.h

```diff
@@ -28,6 +28,14 @@
 #include <sys/un.h>
 #include "stdlog.h"
 
+#ifndef O_CLOEXEC
```

Comment:
Actually, if these flags are not defined then I think rather than defining them to be no-op flags it would be better to call fcntl() to set the close-on-exec flag as a more complete fallback solution.